models.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+
+from app import db 
+
+
+class UserLogin(db.Model):
+
+    id = db.Column(db.Integer, primary_key=True)
+    password = db.Column(db.String(200))
+    google_external_id = db.Column(db.String(200), unique=True) 
```

Comment:
We decided to change this to a boolean field, to show that the user logged in with google. You can call this `google_login` and default value will be `false`. See our google doc for more info.